core/src/io/anuke/mindustry/input/DesktopInput.java

```diff
@@ -429,6 +429,17 @@ void pollInput(){
 
             mode = none;
         }
+
+        if(Core.input.keyTap(Binding.toggle_power_lines)){
+            if(Core.settings.getInt("lasersopacity") == 0){
+                Core.settings.put("lasersopacity", Core.settings.getInt("preferredlaseropacity"));
+            }else{
+                if(!Core.settings.has("preferredlaseropacity")){
```

Comment:
Maybe don't have this condition here, so that the last power opacity gets saved every time you toggle lasers off.